src/D2L.CodeStyle.Analyzers/Immutability/ImmutableDefinitionChecker.cs

```diff
@@ -157,7 +157,7 @@ ImmutabilityContext context
 				return true;
 			}
 
-			var propInfo = GetPropertySyntax(
+			var propInfo = GetPropertyStuff(

```

Comment:
When I was writing it, it started off being just syntax but it grew to be random other stuff too.